packages/grafana-ui/src/components/Card/CardContainer.tsx

```diff
@@ -79,7 +79,7 @@ export const getCardContainerStyles = (
       display: 'grid',
       position: 'relative',
       gridTemplateColumns: 'auto 1fr auto',
-      gridTemplateRows: '1fr auto auto auto',
```

Comment:
@JoaoSilvaGrafana This broke the DS Picker list, because we're giving size to the description, but is not always there:

|`gridTemplateRows: '1fr auto auto auto'`| `gridTemplateRows: 'auto auto 1fr auto'`|
|-|-|
<img width="428" alt="Captura de pantalla 2023-12-13 a las 14 17 27" src="https://github.com/grafana/grafana/assets/5699976/fc579aca-9585-4dcf-838e-13c0633c47e6">|<img width="441" alt="Captura de pantalla 2023-12-13 a las 14 16 59" src="https://github.com/grafana/grafana/assets/5699976/32fe2563-1186-4cbc-8085-3d9704953b39">

Have we tried to not define any `gridTemplateRows` and leave them as `auto`?